t/46-webui-containers.t

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env perl
+# Copyright (C) 2020 SUSE LLC
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, see <http://www.gnu.org/licenses/>.
+
+use Test::Most;
+use Test::Warnings ':report_warnings';
+
+sub clean(){
+    system('docker rm -f gru 2>/dev/null');
+    system('docker rm -f livehandler 2>/dev/null');
+    system('docker rm -f websockets 2>/dev/null');
+    system('docker rm -f scheduler 2>/dev/null');
+    system('docker rm -f webui 2>/dev/null');
+    system('docker rm -f db 2>/dev/null');
+    system('docker network rm testing 2>/dev/null');
+    system('rm -rf workdir 2>/dev/null');
```

Comment:
We could do that in Perl without forking.

Reply:
Won't be necessary because the reason I mentioned with the temp directories Thanks